commit/tokenprice/processor.go

```diff
@@ -3,42 +3,82 @@ package tokenprice
 import (
 	"context"
 	"fmt"
+	"time"
 
 	mapset "github.com/deckarep/golang-set/v2"
+
+	"github.com/smartcontractkit/chainlink-ccip/internal/reader"
+	"github.com/smartcontractkit/chainlink-ccip/pluginconfig"
+	"github.com/smartcontractkit/chainlink-ccip/shared"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
 	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+
+	"github.com/smartcontractkit/libocr/commontypes"
 	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
 
-	"github.com/smartcontractkit/chainlink-ccip/shared"
+	"golang.org/x/exp/maps"
 )
 
 type Processor struct {
```

Comment:
This type doesn't need to be exported, can be private. See https://github.com/smartcontractkit/chainlink-ccip/pull/108#discussion_r1749967753 for more info

Reply:
Not sure I get what you want here. I use it to do `maps.Keys(..)` inside the functions.